packages/application-shell/package.json

```diff
@@ -82,9 +82,7 @@
     "perfume.js": "1.1.4",
     "prop-types": "15.7.2",
     "qss": "2.0.3",
-    "react-modal": "3.8.1",
```

Comment:
this is now in `application-components`, I think it was just a forgotten leftover